### DISCUSSION TITLE
Is there a way to use the `mode` function directly in the colors theme object?

### DISCUSSION THREAD

**Discussion by la55u** [chakra-ui/chakra-ui#2738] [Q&A]
```
// colors.js
export default {
  brand: "#eff9f1",
  brandDark: "#24a66e"
  ...
}
```
This works fine, but in every single component I have to manually tell which color I want to use according to the color mode with the help of `useColorModeValue` hook. This gets tedious very quickly. What I'd like to achieve is to have every color assigned a default value that will be shown when toggling dark/light theme. Something like this:
```
// colors.js
const colors = props => {
  brand: mode("#eff9f1", "#24a66e")(props)
  ...
}
export default colors;
```
This is already the behaviour for example in the [buttons](https://github.com/chakra-ui/chakra-ui/blob/develop/packages/theme/src/components/button.ts) component. I tried to do the same with colors but Chakra doesn't seem to recognize the function there. Any sugessions? Thank you

**Comment by abagshaw** [comment]
See related: https://github.com/chakra-ui/chakra-ui/discussions/2169
